Description: Corrected the place to show up for the desktop entry.
  Add KDE category. KVIrc is not proposed upstream to be a KDE application
  but rather a Qt one. In Debian we build it with KDE dependencies and hence
  we tag application as KDE in for the menu entry.
  .
  This patch can be studied to be forwarded upstream.
Author: Roderick B. Greening <roderick.greening@gmail.com>
Last-Update: 2011-02-23
Origin: vendor, http://hg.debian.org/hg/pkg-kde/kde-extras/kvirc/raw-file/tip/debian/patches/10_fix_desktop_entry.patch
Forwarded: no
---
 cmake/kvirc4.desktop.cmake |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/cmake/kvirc4.desktop.cmake
+++ b/cmake/kvirc4.desktop.cmake
@@ -14,4 +14,4 @@ Exec=${KVIRC_BINARYNAME} -m %u
 Icon=kvirc
 Type=Application
 MimeType=application/x-kva;application/x-kvt;
-Categories=Qt;Network;IRCClient;
+Categories=Qt;KDE;Network;IRCClient;
